modules/tinymce/src/themes/silver/main/ts/Render.ts

```diff
@@ -68,6 +68,9 @@ export interface RenderArgs {
   readonly height: string;
 }
 
+const SpapMessage = 'Try Premium!';
+const SpapLink = 'http://tiny.cloud';
```

Comment:
This URL does not look correct. Are we sure this is what it should be pointed to? At the very least it should be `https://www.tiny.cloud/`. That said though, I'd expect this to point to something highlighting the premium features.

Reply:
No link was provided, so I added the first one available. I wasn't sure if this one should point to anything specific.

Reply:
According to what was said vaguely on the ticket yesterday, it seems this is still TBD... so can you please ensure a follow up JIRA is logged and then add a TODO here?

- [ ] Log follow up Jira to correct this and add a TODO comment referencing the Jira